chain/client/cosmos.go

```diff
@@ -198,6 +219,7 @@ func (c *cosmosClient) broadcastTx(
 			return nil, err
 		}
 
+		//around 750000 - 1500000
 		adjusted := simResponse.GetGasUsed()
 		txf = txf.WithGas(adjusted * 10000)
```

Comment:
we can make this a normal value like `adjusted * 2`

Reply:
In client/tx_factory.go, we already have WithGasAdjustment(1.5) so maybe we just adjust it there. But basically, this doesn't hurt and increase the reliability, thus I just fix what's broken.